src/thunderbots/software/ai/main.cpp

```diff
@@ -84,6 +88,12 @@ int main(int argc, char **argv)
     ros::Subscriber enemy_team_sub = node_handle.subscribe(
         Util::Constants::BACKEND_INPUT_ENEMY_TEAM_TOPIC, 1, enemyTeamUpdateCallback);
 
+    // Initialize the logger
+    std::unique_ptr<g3::LogWorker> logWorker{g3::LogWorker::createLogWorker()};
+    logWorker->addSink(std::make_unique<Util::Logger::RosoutSink>(),
+                       &Util::Logger::RosoutSink::ReceiveLogMessage);
+    g3::initializeLogging(logWorker.get());
```

Comment:
Do we have to go through the process of setting up the logger in basically the same way for `ai`, `backend_input`, `backend_output`? Would be good to just have a function in `util` or something that did this for us if so